src/dmd/backend/bcomplex.d

```diff
@@ -4,14 +4,15 @@
  *
  * Copyright:   public domain
  * License:     public domain
- * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/bcomplex.d, backend/_bcomplex.d)
+ * Source:      $(DMDSRC backend/_bcomplex.d)
```

Comment:
FYI underscore escaping is no longer needed. We enabled the opt-out about two months ago.
See: https://github.com/dlang/dlang.org/pull/2307

Reply:
Also last year you made a point of using absolute links here as they are clickable (see https://github.com/dlang/dmd/pull/7030).
Though I was a fan of DMDSRC, so I don't mind ;-)

Reply:
yes, it should be the full version.